data/migrations/20210802234515_add-episodes-table.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('Episodes', (t) => {
+    t.increments('ID');
```

Comment:
you're using "t.increments("ID")" but in reviewing other migration files, it seems the convention is to do "episodes.increments("ID") or an abbreviation such as "ep.increments("ID")"

Reply:
't' matches conventions from many of the earlier migration files so I think this is acceptable.